sites/skeleton.dev/src/components/examples/framework-components/floating-panel/react/size-constraints.tsx

```diff
@@ -0,0 +1,45 @@
+import { FloatingPanel, Portal } from '@skeletonlabs/skeleton-react';
+import { GripVerticalIcon, MinimizeIcon, XIcon, MinusIcon, MaximizeIcon } from 'lucide-react';
+
+export default function SizeConstraints() {
+	return (
+		<FloatingPanel maxSize={{ width: 900, height: 600 }} minSize={{ width: 300, height: 200 }}>
+			<FloatingPanel.Trigger className="btn preset-filled">Open Panel</FloatingPanel.Trigger>
+			<Portal>
+				<FloatingPanel.Positioner className="z-50">
+					<FloatingPanel.Content>
+						<FloatingPanel.DragTrigger>
+							<FloatingPanel.Header>
+								<FloatingPanel.Title>
+									<GripVerticalIcon className="size-4" />
+									Floating Panel
+								</FloatingPanel.Title>
+								<FloatingPanel.Control>
+									<FloatingPanel.StageTrigger stage="minimized">
+										<MinusIcon className="size-4" />
+									</FloatingPanel.StageTrigger>
+									<FloatingPanel.StageTrigger stage="maximized">
+										<MaximizeIcon className="size-4" />
+									</FloatingPanel.StageTrigger>
+									<FloatingPanel.StageTrigger stage="default">
+										<MinimizeIcon className="size-4" />
+									</FloatingPanel.StageTrigger>
+									<FloatingPanel.CloseTrigger>
+										<XIcon className="size-4" />
+									</FloatingPanel.CloseTrigger>
+								</FloatingPanel.Control>
+							</FloatingPanel.Header>
+						</FloatingPanel.DragTrigger>
+						<FloatingPanel.Body>
+							<p>
+								This is a floating panel that can be dragged, resized, minimized, and maximized. Try dragging from the header or resizing
```

Comment:
Tailor the message to explain what users are seeing. This should detail the min/max pixel size possible for example.